code/datums/martial/krav_maga.dm

```diff
@@ -96,7 +97,8 @@
 				  	"<span class='userdanger'>[A] karate chops your neck, rendering you unable to speak!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
 	D.apply_damage(5, BRUTE)
-	D.silent += 10
+	if(D.silent <= 10)
+		D.silent = Clamp(D.silent + 10, 0, 10)
```

Comment:
You could just set it to 10 then since it can't (or shouldn't) be negative anyway but it's not a big difference.